build.gradle

```diff
@@ -1212,14 +1212,14 @@ if (Integer.parseInt(NUM_COMPILE_THREADS.toString()) < 1) {
     NUM_COMPILE_THREADS = 1
 }
 
-// Check for Gradle 4.8, error if < 4.3.
+// Check for Gradle 4.8, error if < 4.8.
 if (gradle.gradleVersion != "4.8") {
```

Comment:
Would it be easier to have a variable for the minimum gradle version and use that everywhere instead of a "magic number" in 3 places?

Reply:
Yes, this could be done...I will note it for the next time we upgrade the version of gradle that we use (not worth it for this simple bumping of the minimum). Note that we would need two variables, since "current" and "minimum" aren't always the same (could be done similarly to boot JDK version).